src/OpenTelemetry/Metrics/AggregationType.cs

```diff
@@ -62,5 +62,10 @@ internal enum AggregationType
         /// Histogram with sum, count only.
         /// </summary>
         HistogramSumCount = 7,
+
+        /// <summary>
+        /// Histogram with minimum and maximum.
+        /// </summary>
+        HistogramWithMinMax = 8,
```

Comment:
Rework this, main histogram is only sum and count

- Histogram
- HistogramWithBuckets
- HistogramWithBucketsAndMinMax
- HistogramWithMinMax

something like this